tests/__init__.py

```diff
@@ -19,27 +19,32 @@ def init_test_db(app):
                                  attribute_name='Asunnot',
                                  attribute_question='Onko rakennuksessa asunnot?',
```

Comment:
Shoudl attribute_question get the translations for tests too, or do we just test with one language?